### PR TITLE
Added parameters to the query

### PR DESCRIPTION
**Pull request by SpanFish11**

### CODE REVIEW

File: app/delivery/meteor_routes.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ const meteorRouter = express.Router();
 
 meteorRouter.get('/meteors', async (req, res, next) => {
     try {
-        const meteors = await getMeteors();
+        const {date, count, wereDangerousMeteors} = req.query;
+
+        const meteors = await getMeteors(date, count, wereDangerousMeteors);
```
Comment (Mikhail1992) [inline app/delivery/meteor_routes.js:11]: count and wereDangerousMeteors - boolean types but come as strings from request
please convert types

Comment (SpanFish11) [reply app/delivery/meteor_routes.js]: fixed

File: app/delivery/meteor_routes.js [hunk 2]
```diff
@@ -6,7 +6,12 @@ const meteorRouter = express.Router();
 
 meteorRouter.get('/meteors', async (req, res, next) => {
     try {
-        const meteors = await getMeteors();
+        const {date, count, wereDangerousMeteors} = req.query;
+
+        const isDangerous = wereDangerousMeteors === 'true';
+        const hasCount = count === 'true';
+
+        const meteors = await getMeteors(date, hasCount, isDangerous);
```
Comment (Mikhail1992) [inline app/delivery/meteor_routes.js:14]: const meteors = await getMeteors(date, Boolean(count), Boolean(wereDangerousMeteors));

Comment (SpanFish11) [reply app/delivery/meteor_routes.js:14]: But if count=no, then Boolean(count) returns true. Is this the expected behavior?

**Review by Mikhail1992 — APPROVED** [review]

**Review by Mikhail1992 — APPROVED** [review]

**Pull request merged**